main_app/static/css/style.css

```diff
@@ -1,8 +1,49 @@
 body {
-  background-color: #f5f5f5;
+  background-color: rgba(193, 191, 191, 0.808);
   font-family: "Roboto", sans-serif;
+  min-height: 100vh;
+}
+
+.navigation {
```

Comment:
you can remove this so the bootstrap ribbon

Reply:
fixed it